docs/rules/prefer-importing-jest-globals.md

```diff
@@ -42,6 +42,42 @@ describe('foo', () => {
 });
 ```
 
+## Options
+
+This rule can be configured as follows
+
+```json5
```

Comment:
nit: could you make these `json` instead?

I think it's best we do that unless we're putting comments in the block, since ESLint doesn't support JSON5